qiskit_machine_learning/neural_networks/estimator_qnn.py

```diff
@@ -285,4 +302,4 @@ def _backward(
 
                 input_grad, weights_grad = self._backward_postprocess(num_samples, results)
 
-        return input_grad, weights_grad
+        return input_grad, weights_grad
```

Comment:
```suggestion
        return input_grad, weights_grad
        
```
Adding a blank line at the end.